lib/semantics/semantics.cc

```diff
@@ -208,6 +208,28 @@ void SemanticsContext::PopConstruct() {
   constructStack_.pop_back();
 }
 
+void SemanticsContext::ActivateDoVariable(
+    const Symbol *doVariable, const parser::CharBlock location) {
```

Comment:
These `doVariable` arguments should be `const Symbol &` rather than `const Symbol *`. I don't think they are ever null.

Reply:
Thanks, Tim.  I'll make that change.